x/attribute/client/cli/cli_test.go

```diff
@@ -1197,3 +1237,129 @@ func (s *IntegrationTestSuite) TestPaginationWithPageKey() {
 		}
 	})
 }
+
+func (s *IntegrationTestSuite) TestUpdateAccountAttributeExpirationCmd() {
+
+	testCases := []struct {
+		name         string
+		cmd          *cobra.Command
+		args         []string
+		expectErr    string
+		respType     proto.Message
```

Comment:
Me either.  I was going to clean that up.  Thought it might be a good issue to create actually, because they are all over the place.  Also, some places we are just saying expectErr as a `bool` and not testing the error.